src/components/mainTable/MainTable.css

```diff
@@ -0,0 +1,3 @@
+table, th, td {
```

Comment:
it's ok for now. when you will start to use `styled-components` don't forget to use it here as well